content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -22,7 +22,7 @@ This page shows how to enable and configure encryption of secret data at rest.
 The `kube-apiserver` process accepts an argument `--encryption-provider-config`
 that controls how API data is encrypted in etcd.
 The configuration is provided as an API named
-[`EncryptionConfiguration`](/docs/reference/config-api/apiserver-encryption.v1/).
+[`EncryptionConfiguration`](/docs/reference/config-api/apiserver-encryption.v1/). `--encryption-provider-config-automatic-reload` boolean argument determines if the file set by `--encryption-provider-config` should be automatically reloaded if the disk contents change. 
```

Comment:
I'm not sure that detail belongs in a _task_ page. Really, we need a reference page (or section) for the API server component; if we had that, we could definitely as this detail there.

The place where you have added this doesn't feel quite right.

Reply:
While I agree that this would also be part of a reference page, these config knobs are deeply related to each other.  Not having to restart the API server to do key rotation is something that basically all environments would want to do so it feels like that should highlighted with the main config.